1. extraction/gnaf.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+import pickle
+import os
+
+# Accessing the DataFiles Path
+home_dir = os.path.expanduser("~")
+datafiles_path = os.path.join(home_dir, "Desktop", "DataFiles")
+
+# Setting the FilesIn as the source files, and the AppStaging for the pickles
+in_path = os.path.join(datafiles_path, "FilesIn")
+standard_in = os.path.join(in_path, "Standard")
+authority_code_in = os.path.join(in_path, "Authority Code")
+pickle_file_path = os.path.join(datafiles_path, "AppStaging", 'locational-data.pkl')
+files_out_path = os.path.join(datafiles_path, "FilesOut")
```

Comment:
Please don't hard code in the file paths, have these as inputs to the functions instead. Also, avoid using global variables

Reply:
This is code that will only be run once.